cmd/create/machinepool/nodepool.go

```diff
@@ -59,6 +57,21 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		os.Exit(1)
 	}
 
+	// Allow the user to select subnet for a single AZ BYOVPC cluster
```

Comment:
Does this feature is supported only for single AZ clusters?

Reply:
I'd say yes, as single AZ, multi-AZ is somehow deprecated for hosted clusters. As soon as subnets are in the  VPC, customers can create a new NodePool on them.

Hosted ControlPlane is always in HA (MultiAZ) mode.